packages/create-mdx/package.json

```
@@ -1,14 +1,33 @@
 {
   "name": "create-mdx",
-  "description": "Initialize an MDX project",
   "version": "0.16.0",
+  "description": "Initialize an MDX project",
   "license": "MIT",
-  "author": "John Otander",
+  "keywords": [
+    "create",
+    "mdx",
+    "app",
+    "markdown",
+    "react",
+    "jsx",
+    "remark",
+    "unified"
+  ],
+  "homepage": "https://mdxjs.com",
+  "repository": "mdx-js/mdx",
+  "bugs": "https://github.com/mdx-js/mdx/issues",
+  "author": "John Otander <johnotander@gmail.com> (http://johnotander.com)",
+  "contributors": [
+    "John Otander <johnotander@gmail.com> (http://johnotander.com)",
+    "Tim Neutkens <timneutkens@me.com>",
```

This email address is not public / should not be public 😄

```suggestion
    "Tim Neutkens <tim@zeit.co>",
```

---

Hmm, I found it somewhere, that’s why I used it! But fixed!